src/generator.cpp

```diff
@@ -1672,20 +1945,39 @@ void Generator::GeneratorImpl::addImplementationHeaderCode(std::string &code)
     }
 }
 
-void Generator::GeneratorImpl::addLibcellmlVersionCode(std::string &code,
-                                                       bool interface)
+void Generator::GeneratorImpl::addVersionAndLibcellmlVersionCode(std::string &code,
+                                                                 bool interface)
 {
+    std::string versionAndLibcellmlCode;
+
+    if ((interface && !mProfile->interfaceVersionString().empty())
+        || (!interface && !mProfile->implementationVersionString().empty())) {
+        if (interface) {
+            versionAndLibcellmlCode += mProfile->interfaceVersionString();
+        } else {
+            if (modifiedProfile()) {
+                std::regex regEx("([0-9]+\\.[0-9]+\\.[0-9]+)");
+
+                versionAndLibcellmlCode += std::regex_replace(mProfile->implementationVersionString(), regEx, "$1.modified");
```

Comment:
I originally used `post0` because it is part of the definition of version numbering that is understood by Python.  I haven't checked to see if it is part of the semantic definition.  Appending `modified` is not part of that standard.

Reply:
Ok, am going to revert that bit, no problem.

Reply:
Done in commit b1186ee.